data/map.txt

```diff
@@ -26671,12 +26672,14 @@ planet Slylandro
 planet "Smuggler's Den"
 	attributes pirate station "south pirate" south frontier
 	landscape land/loc1
-	description `Smuggler's Den was formerly known as Grendel Station, back when it was an operating refinery. But, it has since been taken over by pirates, who turned it into a haven for all manner of smugglers and privateers. The repair shops here can make a variety of legal and illegal modifications to your ship, and in the shipyard you can buy refurbished starships... if it does not trouble your conscience to think about the likely fates of their previous owners.`
+	description `Smuggler's Den was formerly known as Grendel Station, back when it was an operating refinery. But, it has since been taken over by pirates, who turned it into a haven for all manner of smugglers and privateers. Being a hub of pirate activity, Smuggler's Den has access to fenced goods from beyond the region. The repair shops here can make a variety of legal and illegal modifications to your ship, and in the shipyard you can buy refurbished starships... if it does not trouble your conscience to think about the likely fates of their previous owners. Rumor has it that pilots with the right connections may be able to purchase heavily modified ships here.`
```

Comment:
Fenced goods?

Reply:
That's fine (stolen goods). `Being a hub` is better phrased `As a hub` IMO.

I would probably also join that sentence with the bit about making modifications, and start a new sentence regarding the ships for sale.

Reply:
Made both suggested changes. How does this sound?

`As a hub of pirate activity, Smuggler's Den has access to fenced goods from beyond the region, resulting in repair shops that can make a variety of legal and illegal modifications to your ship. This confluence also provides the opportunity to buy refurbished starships... if it does not trouble your conscience to think about the likely fates of their previous owners.`